Source/Scene/Model.js

```diff
@@ -3088,12 +3098,13 @@ define([
      * @exception {RuntimeError} Failed to load external reference.
      */
     Model.prototype.update = function(frameState) {
-        if (frameState.mode !== SceneMode.SCENE3D) {
+        if (frameState.mode === SceneMode.MORPHING) {
```

Comment:
I imagine implement this will be non-trivial?  It doesn't need to be part of this PR, but we could also fade it out/in.